test/extended/util/annotate/rules.go

```diff
@@ -165,6 +165,7 @@ var (
 			`\[sig-apps\]\[Feature:DeploymentConfig\] deploymentconfigs with test deployments should run a deployment to completion and then scale to zero`,
 			`\[sig-apps\]\[Feature:DeploymentConfig\] deploymentconfigs won't deploy RC with unresolved images when patched with empty image`,
 			`\[sig-apps\]\[Feature:Jobs\] Users should be able to create and run a job in a user project`,
+			`\[sig-imageregistry\]\[Serial\]\[Suite:openshift/registry/serial\] Image signature workflow can push a signed image to openshift registry and verify it`,
```

Comment:
Looks like this list is sorted alphabetically, you should move this skip below, with the other `sig-imageregistry` skips.

Reply:
done